### PR TITLE
running with dan latest vtk-cdat

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

**Comment by scottwittenburg** [comment]
This was just to test the updated build [here](https://github.com/CDAT/conda-recipes/pull/62).

File: vcs/vcsvtk/fillareautils.py [hunk 1]
```diff
@@ -63,7 +63,8 @@ def computeMarkerScale(dataRange, screenGeom, pxScale=None):
     # screenAspect = float(screenGeom[0]) / screenGeom[1]
 
     scale = max(*diffwpoints)
-    if dataAspect > 1:
+    print("DA")
+    if dataAspect < 1:
```
Comment (scottwittenburg) [inline vcs/vcsvtk/fillareautils.py]: @doutriaux1 @danlipsa This is the change that is causing the changes in marker size and the "black screen" (which just really huge markers) on this branch.  This must have slipped in somehow from our experimentation while I was visiting.

As @doutriaux1 and I discussed a couple weeks ago, I'm going to revisit the handling of marker size again soon enough anyway, but I'll push a commit removing this and then create a `vtk_master` branch on the baseline repo and see if we can't get this though so we can really start taking @danlipsa latest VTK build.

**Comment by scottwittenburg** [comment]
Baselines for this PR are located [here](https://github.com/CDAT/uvcdat-testdata/pull/210).

**Comment by scottwittenburg** [comment]
Let's see how this does on circle @doutriaux1 and @danlipsa.  With the new baselines for isofill and streamline, it passed all the tests for me.

**Comment by doutriaux1** [comment]
@scottwittenburg this looks good. I'll need to take Dan's recipe then and build the python 3.7 binaries and switch back to using our channel/nightly then we can merge in master

**Comment by scottwittenburg** [comment]
Yep, sounds good.  Thanks @doutriaux1.

File: vcs/vcsvtk/fillareautils.py (hunk 1 above)
Comment (danlipsa) [reply vcs/vcsvtk/fillareautils.py]: Good you found it. Not sure why I have not seen this. I was testing with master - maybe I did not pull the latest, or was this on a branch?

**Comment by doutriaux1** [comment]
@danlipsa yes I had accidentally pushed it to `vtk_master` branch while experimenting with @scottwittenburg

**Pull request merged**